MigraDoc/code/MigraDoc.RtfRendering/MigraDoc.RtfRendering.csproj

```diff
@@ -2,6 +2,13 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <OutputPath>..\..\..\lib</OutputPath>
+    <Version>1.33.0</Version>
+    <Version Condition="!($(SourceBranchName) == 'master' Or $(SourceBranchName) == 'TableRenderingPerformancePatch') And $(SourceBranchName) != ''">$(Version)-$(SourceBranchName.toLower()).$(BuildNumber)</Version>

```

Comment:
Check this condition is the branch is part of master or not (check with ST)

Reply:
Showed this condition to System team.